### ISSUE TITLE
Multiple changes to same project result in multiple project unloads

### ISSUE THREAD

**Issue by sngbrdb** [tom-englert/ProjectConfigurationManager#10]
Visual Studio 2015
PCM version 1.0.10.0

When multi-pasting values to the same project (for example defined constants for debug x86, x64, and Any CPU), the project will unload and reload for each change.  This seems unnecessary and slows down the process dramatically.

Steps to reproduce:
1. For one project, right-click the defined constants field for one project-configuration-platform.  Pick "Copy"
2. Select multiple defined constants field for other configuration platforms in the same project by holding down the ctrl key and clicking their fields
3.  Right-click the selected fields and pick "Paste"
4.  In the status bar, you'll see the project unload and reload multiple times.  The more fields you pasted into, the slower the update.  Visual Studio is unresponsive while the values are updated.

**Comment by tom-englert** [comment]
I can't reproduce this - if I change many fields, the project is unloaded/saved/loaded only once.
Can you debug this to see if that's a timing issue with the throttle that handles the deferred saving that occurs on only on your machine?

`https://github.com/tom-englert/ProjectConfigurationManager/blob/master/ProjectConfigurationManager.Model/ProjectFile.cs`

`private readonly DispatcherThrottle _deferredSaveThrottle;`

**Comment by sngbrdb** [comment]
I can try... are you asking to set it to log debug output, or to actually debug the extension in Visual Studio?  The latter will take some time; drowning at work at the moment.

**Comment by tom-englert** [comment]
Debug in VS - currently there is no log output at that level.

**Comment by tom-englert** [comment]
I could never reproduce this...
